ansible_tower_client.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
 
   spec.add_runtime_dependency "activesupport"
-  spec.add_runtime_dependency "faraday"
```

Comment:
It looks like v1.0.0 was released a couple days ago, do we need to do anything else to upgrade to it?  Also, if 1.0 is okay, we should probably use `~>1.0`